### ISSUE TITLE
[enhancement] Add inputs validation theme

### ISSUE THREAD

**Issue by equinusocio** [jonathanharrell/hiq#33]
Hi @jonathanharrell What do you think about adding `:valid` and `:invalid` theme for inputs?

I think that them should be applied only to `:required:valid` and `:required:invalid` (with relative focus outline color ) input.

**Comment by jonathanharrell** [comment]
Yes this could work. And we should have a simple variable that the user can override if they want to remove this.

**Comment by jonathanharrell** [comment]
Added in 89aba8253792c1bdbea8677647153ec1d24cdd62.
